crud/select/executor.lua

```diff
@@ -31,52 +30,51 @@ local function scroll_to_after_tuple(gen, param, state, space, scanner)
             return nil
         end
 
-        if scroll_comparator(tuple, scanner.after_tuple) then
+        if scroll_comparator(tuple, after_tuple) then
             return tuple
         end
     end
 end
 
-function executor.execute(plan)
-    local scanner = plan.scanner
+function executor.execute(space, index, scan_value, iter, filter_func, opts)
+    checks('table', 'table', '?table', 'number', 'function', {
+        after_tuple = '?cdata|table',
```

Comment:
Is "cdata" needed here?

tuple is converted to lua table after transferring via netbox.

Reply:
Yes, I just use `space:frommap` in executor tests.

Reply:
BTW, why not? Executor knows nothing about router and other netbox staff.

Reply:
We should not customize the code for tests. We should just write tests that don't require changes in code (if it's not a bug of course ;) )

BTW, frommap():totable() solves your problem.

> BTW, why not? Executor knows nothing about router and other netbox staff.

I've just noticed that tuple as cdata is impossible input for this function.

Reply:
Well, just for you `checks('tuple')` was implemented - https://github.com/tarantool/checks/commit/e4bd1989db6e4e831f7611ddf2d5d20da074b44e

Reply:
It's so cute, I'm waiting for release!

Reply:
See #35.